pineappl_py/docs/source/conf.py

```diff
@@ -1,174 +1,65 @@
+release = pineappl.version
+version = release
 
-# -- General configuration ---------------------------------------------------
 
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
 extensions = [
-    "sphinx.ext.autodoc",
-    "sphinx.ext.doctest",
-    "sphinx.ext.intersphinx",
-    "sphinx.ext.todo",
-    "sphinx.ext.coverage",
-    "sphinx.ext.mathjax",
-    "sphinx.ext.ifconfig",
-    "sphinx.ext.viewcode",
-    "sphinx.ext.autosectionlabel",
-    "sphinx.ext.napoleon",
-    "sphinxcontrib.bibtex",
-    "sphinx.ext.graphviz",
-    "sphinx.ext.extlinks",
+    'sphinx.ext.autodoc',
+    'sphinx.ext.autosectionlabel',
+    'sphinx.ext.extlinks',
+    'sphinx.ext.inheritance_diagram',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.todo',
+    'sphinx_rtd_theme',
 ]
 
-# Add any paths that contain templates here, relative to this directory.
-templates_path = ["_templates"]
-
-# The suffix(es) of source filenames.
-# You can specify multiple suffix as a list of string:
-#
-source_suffix = {
-    ".rst": "restructuredtext",
-    ".txt": "restructuredtext",
-}
-
-autosectionlabel_prefix_document = True
-# autosectionlabel_maxdepth = 10
-# Allow to embed rst syntax in  markdown files.
-enable_eval_rst = True
-
-# The master toctree document.
-master_doc = "index"
-bibtex_bibfiles = ["refs.bib"]
-
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ["shared/*"]
-
-# The name of the Pygments (syntax highlighting) style to use.
-pygments_style = None
-
-# A string to be included at the beginning of all files
-shared = here / "shared"
-rst_prolog = "\n".join([open(x).read() for x in os.scandir(shared)])
 
 extlinks = {
-    "yadism": ("https://nnpdf.github.io/yadism/%s", "yadism - %s"),
     "rustdoc": ("https://docs.rs/pineappl/latest/pineappl/%s", "PineAPPL - %s"),
-    "pineko": ("https://github.com/NNPDF/pineko/%s", "pineko - %s"),
 }
 
+templates_path = ['_templates']
+exclude_patterns = []
+
 # -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-#
-html_theme = "sphinx_rtd_theme"
+html_theme = 'sphinx_rtd_theme'
+html_static_path = ['_static']
 
-# Add any paths that contain custom static files (such as style sheets) here,
-# relative to this directory. They are copied after the builtin static files,
-# so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
 
-# -- Extension configuration -------------------------------------------------
+here = pathlib.Path(__file__).absolute().parent
```

Comment:
This line should also be commented out if it's not used.

Reply:
done in 77995e07